app/controllers/players_controller.rb

```diff
@@ -37,4 +38,14 @@ def create
       redirect_to instance_path(@instance)
     end
   end
+
+  private
+
+  def alert_message(instance)
+    alert = "Lobby not found" # default value since this method gets called from true block of `if @instance.blank?`
+    return alert = "Game already started, try not to miss the next one."      if instance.status == 'ongoing'
+    return alert = "Game already finished, join the next one."                if instance.status == 'done'
+    return alert = "Game is full, tell your buddy to increase more players."  if instance.max_players == Player.where(instance: instance).count
+    alert
+  end
```

Comment:
I was wondering what's this method for?

Reply:
oh derp, I was playing around and thought to use a method but in the end its kinda mendoksai and didn't work, forgot to delete. Let me take care of that now.